File: app/src/main/java/com/github/kongpf8848/animation/activity/tween/TweenTranslateActivity.java
package com.github.kongpf8848.animation.activity.tween;

import android.view.View;

import com.github.kongpf8848.xsdk.ui.activity.BaseActivity;
import com.github.kongpf8848.animation.R;
import com.github.kongpf8848.animation.activity.tween.translate.FidoActivity;
import com.github.kongpf8848.animation.activity.tween.translate.HeartActivity;
import com.github.kongpf8848.animation.activity.tween.translate.ShakeActivity;

import butterknife.OnClick;

/**
 * Created by jack on 2016/9/16.
 */
public class TweenTranslateActivity extends BaseActivity {


    @Override
    protected int getLayoutId() {
        return R.layout.activity_tween_translate;
    }

    @OnClick(R.id.button1)
    public void onButton1(View view) {
        startActivity(ShakeActivity.class);
    }


    @OnClick(R.id.button2)
    public void onButton2(View view) {
        startActivity(HeartActivity.class);
    }

    @OnClick(R.id.button3)
    public void onButton3(View view) {
        startActivity(FidoActivity.class);
    }


}

File: app/src/main/java/com/github/kongpf8848/animation/activity/tween/TweenTranslateActivity.kt
package com.github.kongpf8848.animation.activity.tween

import android.view.View
import butterknife.OnClick
import com.github.kongpf8848.animation.R
import com.github.kongpf8848.animation.base.BaseToolbarActivity
import com.github.kongpf8848.animation.activity.tween.translate.FidoActivity
import com.github.kongpf8848.animation.activity.tween.translate.HeartActivity
import com.github.kongpf8848.animation.activity.tween.translate.ShakeActivity

class TweenTranslateActivity : BaseToolbarActivity() {

    override fun getLayoutId(): Int {
        return R.layout.activity_tween_translate
    }

    @OnClick(R.id.button1)
    fun onButton1(view: View?) {
        startActivity(ShakeActivity::class.java)
    }

    @OnClick(R.id.button2)
    fun onButton2(view: View?) {
        startActivity(HeartActivity::class.java)
    }

    @OnClick(R.id.button3)
    fun onButton3(view: View?) {
        startActivity(FidoActivity::class.java)
    }
}